src/frontend/Sider/Behandling/Stønadsvilkår/PassBarn/PassBarnSaksinformasjonAndreForeldre.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import { styled } from 'styled-components';
+
+import { Alert, VStack } from '@navikt/ds-react';
+
+import { SaksinformasjonAndreForeldre } from '../../../../typer/behandling/behandlingFakta/faktaBarn';
+import { formaterIsoDatoTidKort, formaterIsoPeriode } from '../../../../utils/dato';
+
+const StyledAlert = styled(Alert)`
+    max-width: 50rem;
+`;
+
+const sjekketTidspunkt = (hentetTidspunkt: string) =>
+    ` (sjekket ${formaterIsoDatoTidKort(hentetTidspunkt)})`;
+
+export const PassBarnSaksinformasjonAndreForeldre = ({
+    saksinformasjonAndreForeldre,
+}: {
+    saksinformasjonAndreForeldre: SaksinformasjonAndreForeldre | undefined;
+}) => {
+    // Tidligere behandlinger har ikke informasjonen
+    if (!saksinformasjonAndreForeldre) {
+        return null;
+    }
+    const { hentetTidspunkt, harBehandlingUnderArbeid } = saksinformasjonAndreForeldre;
+    return (
+        <VStack gap={'2'}>
+            {harBehandlingUnderArbeid && (
+                <StyledAlert variant={'warning'} size={'small'}>
+                    <span>
+                        Annen forelder har behandling under arbeid
+                        {sjekketTidspunkt(hentetTidspunkt)}
+                    </span>
+                </StyledAlert>
+            )}
+            <AlertVedtaksperioder saksinformasjonAndreForeldre={saksinformasjonAndreForeldre} />
```

Comment:
Kan man få opp at det både finnes en behandling under arbeid og alerten samtidig?

Hvis ikke så er det unødvnedig å ha en VStack rundt denne. Hvis ja, er det litt meget?

Reply:
Trenger disse å være splittet i to ulike komponenter? Begge tar inn det samme og det er bare 1 case mer å håndtere

Reply:
Ja, man kan få opp de samtidig
Det er 2 ulike komponenter pga if-setning som er tydligere hvis man splitter opp det